ruth/tools/simulator.py

```diff
@@ -381,27 +367,27 @@ def create_route_selection_provider(route_selection: RouteSelectionImpl,
 
 @single_node_simulator.command()
 @click.argument("vehicles_path", type=click.Path(exists=True))
-@click.option("--alternatives", type=click.Choice(AlternativesImpl),
-              default=AlternativesImpl.FASTEST_PATHS,
-              help="Choose an implementation of alternatives [fastest-paths, shortest-paths, distributed]")
+@click.option("--alt-dijkstra-fastest", type=float, default=0.0)
```

Comment:
I'd say that we should just nuke this command-line interface, to avoid updating it all the time, and just use the config file for everything.

Reply:
We were asked to keep the possibility of command line interface.